### DISCUSSION TITLE
Add "Customer total energy added" to "Battery Health dashboard"

### DISCUSSION THREAD

**Discussion by ktsenvolt** [teslamate-org/teslamate#3458] [General]
![image](https://github.com/teslamate-org/teslamate/assets/80360674/fcbd7641-40c2-4e00-be8e-46a85d495432)
![image](https://github.com/teslamate-org/teslamate/assets/80360674/3e01f988-c7cf-4b53-9590-7dad386ca599)
By manually changing the "Total energy from the "ScanMy Tesla" app data it results in more accurate estimated charge cycles.
This is a brand new car (2020 M3 SR+) which I bought in 2020 September 10, I started to use "Teslamate" in July 2021.

**Comment by jheredianet** [comment]
Hi, It's possible to add that option, but it wouldn't do much good. Everything that is tracked on Teslamate is shown here. What it means is not exactly how many charges cycles the battery actually has, but how many charge cycles have been recorded in Teslamate. If you change that data manually, it will make the other calculations inconsistent, unlike the range and capacity that only serve as a reference.

Have in mind that all these data are estimations based on the database, they are not a reading from ODB (like ScanMy Tesla and other tools do)
